src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -373,3 +374,150 @@ message Version {
   // incremented if some job or task entity is created/updated.
   // uint64 revision = 2;
 }
+
+message SharedContainerVolumeSource {
```

Comment:
I think volume model should be in a separate `volumes.proto`. titus_base is for very basic/fundamental types.

Reply:
extracted to `titus_volumes.proto`.